docs/development/contribution-documentation/documentation-process-internal-contributor/README.md

```diff
@@ -33,14 +33,14 @@ In the modal window select the repository ""*opf/openproject*"". Also select a f
 
 > **Important**: Before you make changes always update your local repository.
 
-1. Select the branch you want to work on, e.g.`release/12.1` in the main toolbar.
+1. Select the branch you want to work on, e.g. `release/12.1` in the main toolbar.
```

Comment:
Hi @cbliard how comes you have an older version on this?
I was working on the docs and this will cause merge conflicts
please check here:
https://www.openproject-edge.com/docs/development/contribution-documentation/documentation-process/